service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -381,6 +423,35 @@ private void runAzureDatabaseUtils(
           workspaceId,
           namespace);
       deleteContainer(aksApi, safePodName, namespace);
+      deleteSecret(aksApi, safePodName, namespace);
+    }
+  }
+
+  private void createSecret(
+      CoreV1Api aksApi, Map<String, String> secretStringData, String secretName, String namespace)
+      throws ApiException {
+
+    final V1Secret secretDefinition = new V1Secret().metadata(new V1ObjectMeta().name(secretName));
+    secretStringData.forEach(secretDefinition::putStringDataItem);
+
+    try {
+      logger.info("Creating secret {}", secretDefinition);
```

Comment:
Are there any security concerns around logging the secret? Maybe we could just log the metadata and not the data which presumably contains the actual secret...

Reply:
I'm not 100% sure, but I'll take it out. better safe than sorry